prover/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+cd ..
+export ZKSYNC_HOME=$(pwd)
+cd prover
+
+sed -i '' 's/^ETH_SENDER_SENDER_PROOF_SENDING_MODE.*$/ETH_SENDER_SENDER_PROOF_SENDING_MODE=OnlyRealProofs/' ../etc/env/dev.env
+sed -i '' 's/^ETH_SENDER_SENDER_PROOF_LOADING_MODE.*$/ETH_SENDER_SENDER_PROOF_LOADING_MODE=FriProofFromGcs/' ../etc/env/dev.env
+sed -i '' 's/^FRI_PROVER_SETUP_DATA_PATH.*$/FRI_PROVER_SETUP_DATA_PATH=vk_setup_data_generator_server_fri\/data\//' ../etc/env/dev.env
+sed -i '' 's/^FRI_PROOF_COMPRESSOR_UNIVERSAL_SETUP_PATH.*$/FRI_PROOF_COMPRESSOR_UNIVERSAL_SETUP_PATH=..\/keys\/setup\/setup_2^26.key/' ../etc/env/dev.env
+sed -i '' 's/^FRI_PROOF_COMPRESSOR_PROMETHEUS_PUSH_INTERVAL_MS.*$/FRI_PROOF_COMPRESSOR_PROMETHEUS_PUSH_INTERVAL_MS=100000000/' ../etc/env/dev.env
```

Comment:
we have fixed the underlying library that was spamming the logs.
So when you sync your boojum-integration branch past https://github.com/matter-labs/zksync-era/pull/253 - the logs should not show up anymore

Reply:
Cool, i remove the prometheus push internal change here [478d130](https://github.com/matter-labs/zksync-era/pull/258/commits/478d130dec5f1707cf71c9c82f46a8db2bd67393)